core/src/jvmMain/kotlin/com/juul/kable/logs/SystemLogEngine.kt

```diff
@@ -0,0 +1,30 @@
+package com.juul.kable.logs
+
+import com.juul.kable.jvmNotImplementedException
+
+public actual object SystemLogEngine : LogEngine {
+
+    override fun verbose(throwable: Throwable?, tag: String, message: String) {
+        debug(throwable, tag, message)
+    }
```

Comment:
Particular reason for this function to call to `debug` but the others to call `jvmNotImplementedException()`?

Reply:
No, this slipped away. Fixing it.